tst/EnergyPlus/unit/ReportSizingManager.unit.cc

```diff
@@ -353,3 +353,25 @@ TEST_F( EnergyPlusFixture, ReportSizingManager_RequestSizingZone ) {
 	ZoneSizingInput.deallocate();
 
 }
+
+TEST_F( EnergyPlusFixture, ReportSizingManager_ReportSizingOutputTest ) {
+	// issue #6112
+	std::string CompName;
+	std::string CompType;
+	std::string VarDesc;
+	std::string UsrDesc;
+	Real64 VarValue;
+	Real64 UsrValue;
+
+	// input values
+	CompType = "BOILER:HOTWATER";
+	CompName = "RESIDENTIAL BOILER ELECTRIC";
+	VarDesc = "Design Size Nominal Capacity[W]";
+	VarValue = 105977.98934;
+	UsrDesc = "User-Specified Nominal Capacity[W]";
+	UsrValue = 26352.97405;
+
+	// boiler hot water autosizing and userspecified nominal capacity reporting
+	ReportSizingManager::ReportSizingOutput( CompType, CompName, VarDesc, VarValue, UsrDesc, UsrValue );
```

Comment:
This unit test doesn't really check anything.  Perhaps you can follow the patterns in SQLite.unit.cc to actually test what was written?